### PR TITLE
CXEC-43183 new pss Nuvei integation

### PR DESCRIPTION
**Pull request by zeyun123**
hello
new implementation  of psp to our OPF WB : Nuvei
https://jira.tools.sap/browse/CXEC-43183

### CODE REVIEW

File: postman/nuvei/Hosted-Fields/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+## Introduction ##
+This Postman Collection aids in integrating [Nuvei](https://docs.nuvei.com/documentation/home/) into the Open Payment Framework (OPF).
+
+The integration supports:
+
+* Authorize card using Hosted Filed
```
Comment (opf-postman) [inline postman/nuvei/Hosted-Fields/README.md:6]: typo->Field

File: postman/nuvei/Hosted-Fields/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+* ``integrationId`` maps to ``accountGroupId`` in postman
+* ``configurationId`` maps to ``accountId`` in postman
+
+**4. merchantID and siteID**
+
+You already gets those IDs from the Nuvei backoffice.(under Settings -> My Integrations Settings.)
+
+**5. secretKey**
+
+Same place that you find the merchantId, you need click the "Reveal" button to get the secret Key.
```
Comment (opf-postman) [inline postman/nuvei/Hosted-Fields/README.md:77]: need to

Comment (opf-postman) [inline postman/nuvei/Hosted-Fields/README.md:77]: Same place->The same place

**Review by opf-postman — COMMENTED** [review]
some minor comments.

**Review by rmcotton-sap — COMMENTED** [review]
confirming this is https://opf-iss-d0.opf.commerce.stage.context.cloud.sap/opf-workbench/settings/payment/integration/505 the relating config?

File: postman/nuvei/Hosted-Fields/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+This Postman Collection aids in integrating [Nuvei](https://docs.nuvei.com/documentation/home/) into the Open Payment Framework (OPF).
+
+The integration supports:
+
+* Authorize card using Hosted Filed
+* Settlement
+* Refund
+* Reversal
+* Reauthorization
+
```
Comment (rmcotton-sap) [inline postman/nuvei/Hosted-Fields/README.md:11]: Add planned backlog items section and call out 3d secure

File: postman/nuvei/Hosted-Fields/environment_configuration.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
+{
```
Comment (rmcotton-sap) [inline postman/nuvei/Hosted-Fields/environment_configuration.json:1]: there is no authentication on the notifications

**Comment by zeyun123** [comment]
confirming this is https://opf-iss-d0.opf.commerce.stage.context.cloud.sap/opf-workbench/settings/payment/integration/505 the relating config?
---> hello ,  505 is the payment method way which is easier for me to config the integration and test the result (because only one gateway per site , some one may change the gateway while I am doing the configuration) . 
and everything is ok ,then I transfer to gateway way to commit the codes :)

File: postman/nuvei/Hosted-Fields/environment_configuration.json (hunk 4 above)
Comment (zeyun123) [reply postman/nuvei/Hosted-Fields/environment_configuration.json:1]: yes,  I don't find any thing about the authentication on the notification also from the document:
https://docs.nuvei.com/documentation/integration/webhooks/

Comment (rmcotton-sap) [reply postman/nuvei/Hosted-Fields/environment_configuration.json:1]: There is an ip allowlisting

https://docs.nuvei.com/documentation/integration/webhooks/#additional-integration-steps

**Pull request merged**